src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseDataFileServiceTest.cs

```diff
@@ -1275,50 +1252,49 @@ public async Task GetInfo_MatchingSourceZipBlob()
         {
             var release = new Release();
 
-            var zipFile = new File
+            var zipReleaseFile = new ReleaseFile
             {
-                RootPath = Guid.NewGuid(),
-                Filename = "test-data-archive.zip",
-                Type = DataZip
+                Release = release,
+                File = new File
+                {
+                    RootPath = Guid.NewGuid(),
+                    Filename = "test-data-archive.zip",
+                    Type = DataZip
+                }
             };
-            var dataFile = new File
+            var dataReleaseFile = new ReleaseFile
             {
-                RootPath = Guid.NewGuid(),
-                Filename = "test-data.csv",
-                Type = FileType.Data,
-                Source = zipFile,
-                Created = DateTime.UtcNow,
-                CreatedById = _user.Id
+                Release = release,
+                Name = "Test data",
+                File = new File
+                {
+                    RootPath = Guid.NewGuid(),
+                    Filename = "test-data.csv",
+                    Type = FileType.Data,
+                    Source = zipReleaseFile.File,
+                    Created = DateTime.UtcNow,
+                    CreatedById = _user.Id
+                }
+            };
+            var metaReleaseFile = new ReleaseFile
+            {
+                Release = release,
+                File = new File
+                {
+                    RootPath = Guid.NewGuid(),
+                    Filename = "test-data.meta.csv",
+                    Type = Metadata,
+                    Source = zipReleaseFile.File,
+                }
             };
 
             var contentDbContextId = Guid.NewGuid().ToString();
-
             await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
             {
                 await contentDbContext.AddRangeAsync(
-                    new ReleaseFile
-                    {
-                        Release = release,
-                        File = dataFile
-                    },
-                    new ReleaseFile
-                    {
-                        Release = release,
-                        File = zipFile
```

Comment:
This was actually wrong before just in case your wondering. Zip files don't have a ReleaseFile (and I'm not sure why they need to exist as a File at all!)